Slim/Networking/Async/HTTP.pm

```diff
@@ -89,9 +89,25 @@ sub new_socket {
 		);
 	}
 	
+	my %args = @_;
+	my %socks = ();
+	
+	if ($self->socksAddr) {
+		require Slim::Networking::Async::Socket::HTTPSocks;
+		require Slim::Networking::Async::Socket::HTTPSSocks;
+		%socks = ( 	ProxyAddr => $self->socksAddr,
```

Comment:
The whitespace terror! Please use tabs for main indentation of the code, but spaces to align stuff in a block...